chart/templates/dropdb.yaml

```diff
@@ -37,9 +37,9 @@ spec:
         - name: PGPORT
           value: "{{ .Values.application.database_port }}"
         - name: PGUSER
-          value: {{ .Values.application.database_user }}
+          value: {{ .Values.application.database_user | b64dec }}
         - name: PGPASSWORD
-          value: {{ .Values.application.database_password }}
+          value: {{ .Values.application.database_password | b64dec }}
```

Comment:
Does this job have access to the secrets?

Reply:
No, the job is called after the release has been deleted and the secret is part of the release. So indeed, for the few seconds the job is running the password appears in cleartext in its yaml file.